burotel/indico_burotel/plugin.py

```diff
@@ -97,6 +98,8 @@ def _before_request(self):
     def _override_templates(self, sender, **kwargs):
         return os.path.join(self.root_path, 'template_overrides')
 
-    def _inject_long_term_attribute(self, rooms):
-        for room in rooms:
-            room['is_long_term'] = Room.get(room['id']).get_attribute_value('long-term', False)
+    def _inject_long_term_attribute(self, sender, **kwargs):
+        if sender == RoomSchema:
+            long_term_room_ids = {r[0].id for r in Room.find_with_attribute('long-term')}
```

Comment:
i guess this gives you `attribute, value` tuples? if yes, `r` is misleading. also, check the attribute value (e.g. if it;s `'true'`) - otherwise the presence of the attribute makes the room long-term which is strange

Reply:
Ah true, leftover from indico shell